Pod/INSImageView.swift

```diff
@@ -41,10 +41,15 @@ public class INSImageView: UIImageView {
     }
     
     public required init?(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
+        super.init(coder: aDecoder)
+        setup()
     }
     
     private func setup() {
+        if imageView.image == nil {
+            imageView.image = super.image
+        }
+        super.image = nil
```

Comment:
Strange, I tested it and it worked fine in storyboard. I'll try get some UI tests up.